packages/config/src/enums.ts

```diff
@@ -0,0 +1,109 @@
+export enum StoreStatus {
+  live = 'live',
+  disabled = 'disabled',
+  stopped = 'stopped'
+}
+
+export enum Feature {
+  autocomplete = 'autocomplete',
+  search = 'search',
+  recommendation = 'recommendation',
+  content = 'content',
+  custom = 'custom',
+}
+
+export enum Platform {
+  generic = 'generic',
+  shopify = 'shopify',
+  bigcommerce = 'bigcommerce',
+  magento = 'magento'
+}
+
+export enum AnalyticsEventKey {
+  viewPage = 'view-page',
```

Comment:
@Bitaru do we need to have `add-to-cart`, `click-item` and other click events here?

Reply:
click-item is there, `add-to-cart` has been added